routes/auth-routes.js

```diff
@@ -52,7 +46,7 @@ router.post('/verif', function (req, res) {
         //updating the user table with user verification code
         else {
           if (!user.rowCount) {
-            res.redirect('/auth/verifAgain');
+            res.redirect('http://localhost:3000/Activation');
```

Comment:
Why have you added the domain to this URL (http://localhost:3000)? Once this moves to its own domain e.g. http://convenient.com, would this cause a problem?